src/commands/dockersupport/localDockerSupport.ts

```diff
@@ -1,8 +1,3 @@
-/*---------------------------------------------------------------------------------------------
```

Comment:
Looks like you accidentally removed the Copywrite comment

Reply:
just added it back in!